src/gui/torrentcontentmodel.cpp

```diff
@@ -50,6 +53,11 @@
 #include "torrentcontentmodelfolder.h"
 #include "torrentcontentmodelfile.h"
 
+#ifdef Q_OS_MAC
+struct NSImage;
+QPixmap qt_mac_toQPixmap(const NSImage *image, const QSizeF &size);
```

Comment:
It is a private function from QtGui. Yes, it is exported due to the fact plugins need it. There is no reason in reimplementing it.